MTK - 55.7.0 release branch (#5081)
EnterpriseDB/docs: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx

---
title: "Migration Toolkit 55.7.0 release notes"
navTitle: Version 55.7.0
---

Released: 12 Dec 2023

New features, enhancements, bug fixes, and other changes in Migration Toolkit 55.7.0 include:

|     Type     |                                                                                                                                                                                         Description                                                                                                                                                                                         |
| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | Enhancement  | The Migration Toolkit is now certified to work with all the BigAnimal supported cluster types including the distributed high availability clusters. |

Reviewer: ```suggestion
| Enhancement  | The Migration Toolkit is now certified to work with all the BigAnimal supported cluster types including the distributed high-availability clusters. |
```

Author: ok @mlewandowski-edb.

| Enhancement  | Improved the performance of table-level parallel loading of many tables by updating the operation to fetch all the constraints in a single network round trip. [Support ticket #97379] |

> | Bug&nbsp;fix | Fixed an issue where user created triggers on the target PostgreSQL or EDB Postgres Advanced Server databases were getting disabled during the data migration phase. [Support ticket #97637] |

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue where user-created triggers on the target PostgreSQL or EDB Postgres Advanced Server databases were getting disabled during the data migration phase. [Support ticket #97637] |
```

> | Bug&nbsp;fix | Fixed an issue that caused re-evaluation of the same table with an indefinite loop for table-level parallel loading mode. [Support ticket #96054]|

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue that caused reevaluation of the same table with an indefinite loop for table-level parallel loading mode. [Support ticket #96054]|
```

| Bug&nbsp;fix | Fixed an issue where data copy was failing while migrating from a source Oracle database configured with the Latin9 character set. [Support ticket #99475, #99506] |
| Bug&nbsp;fix | Fixed an issue where the copy API was failing to copy the data and a subsequent attempt to copy the data with the bulk insert mode was failing with a duplicate error in the target database.[Support ticket #99506, #97784] |

> | Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source SQL Server table with a case sensitive column name in the EDB Postgres Advanced server target database. |

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source SQL server table with a case-sensitive column name in the EDB Postgres Advanced server target database. |
```

> | Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source MySQL table due to a case sensitive table name. |

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source MySQL table due to a case-sensitive table name. |
```

> | Bug&nbsp;fix | Fixed an issue for partitioned tables with a SUBPARTITION TEMPLATE clause that caused a partitioned table sub-partitioned by range to fail to migrate. |

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue for partitioned tables with a `SUBPARTITION TEMPLATE` clause that caused a partitioned table sub-partitioned by range to fail to migrate. |
```

> | Bug&nbsp;fix | Fixed an issue where the ``capture_admin`` database system group was getting included when the ``-allgroups`` option was specified for migration from the source EDB Postgres Advanced Server version 16 database. |

Reviewer: ```suggestion
| Bug&nbsp;fix | Fixed an issue where the `capture_admin` database system group was getting included when the `-allgroups` option was specified for migration from the source EDB Postgres Advanced Server version 16 database. |
```
